### PR TITLE
Update input for MassInflow boundary condition 

### PR DESCRIPTION
**Pull request by malhardp333**
* Currently, the `MassInflow` boundary condition requires `totalTemperatureRatio` in the calculations, but it is not set as required input in the Python API. 
* Hence, it has been added as a required input for this boundary condition, and the corresponding tests have been updated as part of this PR.

### CODE REVIEW

File: flow360/component/flow360_params/boundaries.py [hunk 1]
```diff
@@ -234,6 +234,7 @@ class MassInflow(BoundaryWithTurbulenceQuantities):
 
     type: Literal["MassInflow"] = pd.Field("MassInflow", const=True)
     mass_flow_rate: PositiveFloat = pd.Field(alias="massFlowRate")
+    total_temperature_ratio: PositiveFloat = pd.Field(alias="totalTemperatureRatio")
```
Comment (benflexcompute) [inline flow360/component/flow360_params/boundaries.py:237]: is there a range for this? Or is positive enough?

**Comment by benflexcompute** [comment]
@yifan-flex Let's add a memo to incorporate this into Simulation as well.

File: flow360/component/flow360_params/boundaries.py (hunk 1 above)
Comment (malhardp333) [reply flow360/component/flow360_params/boundaries.py:237]: Just positive is enough. In the formula where totalTemperatureRatio gets used, I don't see any other restriction which limits its range.

**Review by benflexcompute — COMMENTED** [review]
Can you take a look at the unit test:
`tests.test_updater.py::test_updater_from_files`
and add the MassInflow into one of the solver JSON and make sure the totalTermeratureRatio is properly handled by legacy updater and the new model?

**Comment by yifan-flex** [comment]
> @yifan-flex Let's add a memo to incorporate this into Simulation as well.

It has already been incorporated

**Review by benflexcompute — APPROVED** [review]
